refocus-bdk-client.js

```diff
@@ -715,7 +716,8 @@ module.exports = (config) => {
       const events = {
         log: msg,
         roomId: room,
-        actionType: type
+        actionType: type,
+        botId: botName
```

Comment:
bot name is tricky "promoted" to botId? Is that the idea?

Reply:
its called botid but everywhere we use it we are putting botname in